doc/api/http.markdown

```diff
@@ -966,6 +966,8 @@ Example:
 
     http.get("http://www.google.com/index.html", function(res) {
       console.log("Got response: " + res.statusCode);
+      // consume response body
+      res.resume();
```

Comment:
I don't think that's necessary, `statusCode` is a property that exists when the function is called and isn't attached to the streaming nature of `res` in any way (beyond this point at least). I'm more concerned with the lack lack of information on what `res.resume()` does but explaining it here, even "enter flowing mode", is going to detract from the point of the example and likely take up too much additional space.